apps/core-app/src/main/utils/app-root-path.ts

```
@@ -0,0 +1,200 @@
+export type DevDataMigrationStatus =
+  | 'skipped-packaged'
+  | 'skipped-marker-exists'
+  | 'skipped-source-missing'
+  | 'skipped-target-not-empty'
+  | 'skipped-same-path'
+  | 'migrated'
+  | 'failed'
+
+export interface AppPathLike {
+  isPackaged: boolean
+  getPath(name: 'userData'): string
```

ESLint

[error] Shorthand method signature is forbidden. Use a function property instead.

(ts/method-signature-style)